resources/auth/model.js

```diff
@@ -19,3 +19,13 @@ exports.filter = filter => {
     .where(filter)
     .first();
 };
+
+// FindBy is used when comparing passwords from the user on login
+// And password stored in the db
+
+exports.findBy = param => {
+  return db('users')
+    .select('full_name', 'email', 'password', 'isConfirmed')
+    .where(param)
+    .first();
+};
```

Comment:
This looks a bit like what the filter function does?

Reply:
Yeah I know, I just needed to use the password in order to be able to compare the passwords on login, and I wasn't sure if it was a good idea to return the password after signup?

Reply:
Right, yeah, sorry.

Didn't spot that at first glance, necessary to return the password for comparison.

Nice one